src/Components/Web/src/Microsoft.AspNetCore.Components.Web.csproj

```diff
@@ -7,6 +7,8 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <RootNamespace>Microsoft.AspNetCore.Components</RootNamespace>
     <Nullable>enable</Nullable>
+    <!-- SYSLIB0020: JsonSerializerOptions.IgnoreNullValues is obsolete -->
+    <NoWarn>$(NoWarn);SYSLIB0020</NoWarn>
```

Comment:
Why does this warning happen? I can't see any usage of `IgnoreNullValues` in this PR.

Reply:
This warning was coming from generated code, but it looks like that's not happening anymore for some reason. I've just removed this.